run/system_package/e2e_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/cloudrunci"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestDiagramService(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+	service := cloudrunci.NewService("diagram", tc.ProjectID)
+	if err := service.Deploy(); err != nil {
+		t.Fatalf("could not deploy %s: %v", service.Name, err)
+	}
+	defer service.Clean()
+
+	requestPath := "/diagram.png?" + url.QueryEscape("digraph G { A -> {B, C, D} -> {F} }")
+	req, err := service.NewRequest("GET", requestPath)
+	if err != nil {
+		t.Errorf("service.NewRequest: %q", err)
+		return
+	}
+	resp, err := http.DefaultClient.Do(req)
```

Comment:
Might want to create a client and set a timeout. If everything is working, it should be fast. So, a short timeout seems reasonable (e.g. 5-10 seconds).